airbyte-integrations/bases/base-normalization/integration_tests/resources/test_simple_streams/data_input/catalog.json

```diff
@@ -124,10 +124,10 @@
         },
         "supported_sync_modes": ["full_refresh", "incremental"],
         "source_defined_cursor": true,
-        "default_cursor_field": []
+        "default_cursor_field": ["_ab_cdc_updated_at"]
```

Comment:
this stream is weird and doesn't have an actual lsn field. Use updated_at.

(I _think_ this stream isn't even used as-is; it has no records in messages.txt)

Reply:
💪